public/scripts/tagsList.js

```diff
@@ -0,0 +1,30 @@
+function getTagObject(content){
+    let tagElement = document.createElement('div')
+    tagElement.classList.add('tagElement')
+
+    let tagElementText = document.createElement('span')
+    let text = document.createTextNode(content)
+    tagElementText.classList.add('tagElementText')
+    tagElementText.appendChild(text)
+
+    let deleteTag = document.createElement('button')
+    deleteTag.value = "X"
+
+    tagElement.appendChild(tagElementText)
+    tagElement.appendChild(deleteTag)
+
+    return tagElement
+
+}
+(function(){
+    console.log('wow')
```

Comment:
😮

Reply:
Usually it's a good idea to delete console logs.

Reply:
I forgot to remove some console logs :D